tests/libvirt-environment/Justfile

```diff
@@ -0,0 +1,56 @@
+# This file can be used with `just` (https://github.com/casey/just)
+ 
+TAG = 'smoke-test-installer'
+NET = 'bridge'
+QEMU_IMG_PATH_RHCOS = "/opt/app-root/src/qemu-img/rhcos-qemu.qcow2"
+SSH_KEY_PATH = "/root/.ssh"
+
+dl-rhcos src='' outputfile='rhcos-qemu.qcow2':
+	#!/usr/bin/env bash
+	src={{src}}
+	if [ -z $src ]; then
+	    echo "SRC1: $src"
+	    build=$(curl -s https://releases-rhcos.svc.ci.openshift.org/storage/releases/maipo/builds.json | jq -r '.builds[0]')
```

Comment:
Can you use the long flags in all of these commands? I think they are more descriptive.